OpenOversight/app/models/database.py

```diff
@@ -620,6 +591,211 @@ class Image(BaseModel, TrackUpdates):
 )
 
 
+class Incident(BaseModel, TrackUpdates):
```

Comment:
Classes were reordered so that I could accurately label return values for the `Department` class.